app/abilities/page.js

```diff
@@ -0,0 +1,6 @@
+import { not } from "@ember/object/computed";
+import { Ability } from "ember-can";
+
+export default Ability.extend({
```

Comment:
Class syntax

Reply:
Is it compatible with Ember version 3.12?
We had a similar discussion in another PR
https://github.com/adfinis-sygroup/timed-frontend/pull/562#discussion_r657992139

Reply:
In other words the update was not done in another PR. Nice.

Reply:
Resolved because update didn't happen